transit_odp/browse/templates/browse/operators/operator_detail.html

```diff
@@ -238,6 +238,16 @@ <h3 class="govuk-heading-m govuk-!-margin-top-7">
             </div>
           {% endif %}
         </div>
+        <div class="govuk-grid-column-one-half">
+          <div class="feed-stat">
+            <span class="feed-stat__value">{{ avl_services_require_attention_percentage }}%</span>
+            <span class="feed-stat__label">
+              {% filter force_escape %}
+                {% trans "Services requiring attention" %}
+              {% endfilter %}
+            </span>
+          </div>
+        </div>
```

Comment:
@amitkpmguk 

There is another report link to be added under 'Browse Location data' (same link as 'Download detailed export' under Timetables data).

Reply:
@ndemenezes , Yes, we have a separate ticket for that, which covers link addition as well as adding new columns in the CSV

Reply:
@ndemenezes , Sorry misread the story, I have added the required link